update-local-tap.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TAP_NAME="aws/homebrew-tap"
```

Comment:
Curious, isn't tap name is `aws/tap`? Is it different when running locally?

Reply:
Homebrew will automatically add `homebrew-` in front of the name in the command `brew tap` but the folder and location on disk is still the full repo name. 
Added comments to explain this behavior.